modules/virt-configuring-highburst-profile.adoc

```diff
@@ -0,0 +1,19 @@
+
+// Module included in the following assemblies:
+//
+// * virt/advanced_vm_management/virt-vm-control-plane-tuning.adoc
+
+:_content-type: PROCEDURE
+[id="virt-configuring-highburst-profile_{context}"]
+= Configuring a highBurst profile
+
+Use the `highBurst` profile to create and maintain a large number of virtual machines (VMs) in one cluster.
+
+.Procedure
+
+. Apply the following patch to enable the `highBurst` `tuningPolicy` profile.
++
+[source,terminal]
+----
+$ kubectl patch -n openshift-cnv hco kubevirt-hyperconverged --type=json -p='[{"op": "add", "path": "/spec/tuningPolicy", "value": "highBurst"}]'
```

Comment:
"kubectl" is upstream. Needs to be "oc". 

I also recommending adding line breaks with backslashes so that the user does not have to scroll to see the whole command.